vault/src/core/vault_format.rs

```
@@ -0,0 +1,27 @@
+use crate::core::models::{Credential, Vault};
+
+pub fn createEmptyVault () ->Vault{
+    Vault{
+        entries: Vec::new(),
+    }
+}
+
+pub fn addEntry (vault: &mut Vault, service: String, username: String, password: String) -> {
```

Clippy (1.93.1)

[error] expected type, found `{`

(error)